docs/src/main/asciidoc/rest-data-panache.adoc

```diff
@@ -54,6 +54,39 @@ include::./status-include.adoc[]
 * Implement the Panache entities and/or repositories as explained in the xref:hibernate-orm-panache.adoc[Hibernate ORM with Panache guide].
 * Define the interfaces for generation as explained in the resource generation section.
 
+=== Hibernate Reactive
+
+* Add the required dependencies to your `pom.xml`
+** Hibernate Reactive REST Data with Panache extension (`quarkus-hibernate-reactive-rest-data-panache`)
+** A JDBC driver extension (`quarkus-reactive-pg-client`, `quarkus-reactive-mysql-client`, ...)
```

Comment:
These are not JDBC drivers, they are Vert.x reactive database drivers

Reply:
Fixed